Sources/NIOHTTP2/HTTP2ChannelHandler+InlineStreamMultiplexer.swift

```diff
@@ -167,11 +167,15 @@ extension NIOHTTP2Handler {
     public struct StreamMultiplexer: @unchecked Sendable {
         // '@unchecked Sendable' because this state is not intrinsically `Sendable`
         // but it is only accessed in `createStreamChannel` which executes the work on the right event loop
-        private let inlineStreamMultiplexer: InlineStreamMultiplexer
+
+        private let inlineStreamMultiplexer: NIOLoopBound<InlineStreamMultiplexer>
```

Comment:
I don't think `NIOLoopBound` is the right approach here. There isn't a particular requirement to force the event loop dance out this far, I don't think. In practice we're calling an API that is thread-safe inside `InlineStreamMultiplexer`, so I wonder if using a lens type might be better.

Reply:
I have made `InlineStreamMultiplexerSendableView` for this purpose